cuebot/src/main/resources/conf/ddl/postgres/migrations/V3__Increase_show_length.sql

```diff
@@ -0,0 +1,118 @@
+-- Increase the length of show names
```

Comment:
Could you add some more comment text explaining what's going on here? I.e. why it's necessary to drop and recreate the views. I think it will be helpful for folks in the future.